src/_03_More_Array_Fun/MoreArrayFunTest.java

```diff
@@ -0,0 +1,110 @@
+package _03_More_Array_Fun;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class MoreArrayFunTest {
+
+	// ARRAY STRING PRINTER
+
+	@Test
+	public void arrayPrinterWorks_fourStrings() {
+		String[] array = { "blue", "cello", "cactus", "electric" };
+		String allValues = MoreArrayFun.stringArrayPrinter(array);
+		assertEquals("blue, cello, cactus, electric", allValues);
+
+	}
+
+	@Test
+	public void arrayPrinterWorks_twoStringsWithSpaces() {
+
+		String[] arrayTwo = { "snow leopard enclosure", "bean paste candy" };
```

Comment:
This probably doesn't need to be called `arrayTwo`.  `array` would work fine.

Reply:
👍